sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -204,11 +205,14 @@ def get_outputs(op_ix):
         else:
           # Otherwise serialize the source and execute it there.
           # TODO: Use SDFs with an initial impulse.
+          # The Java runner harness strips the base64 encoding. do the same
```

Comment:
Ah, so it'll all work out once we start publishing runner-proto encoded values in our submitted graphs.

Technically it'd be nice to publish a different URN for this. (Also, interestingly the stripping is done for sources but not for DoFns.)